src/posix/tty.rs

```diff
@@ -844,7 +845,34 @@ fn udev_hex_property_as_u16(d: &libudev::Device, key: &str) -> ::Result<u16> {
     }
 }
 
-#[cfg(all(target_os = "linux", not(target_env = "musl")))]
+// FIXME: this hack is only to simulate Bluegiga device on Thin Client NOT to be used in production! 
```

Comment:
We cannot approve this PR with this hack in it.

Reply:
@erik-nymi : I updated the comment, the comment that contained the word "hack" was a legacy from when there was no option to choose the workaround at compile-time.